person.rb

```diff
@@ -2,14 +2,19 @@
 
 class Person
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
     @corrector = Corrector.new
+    @rentals = []
+  end
+
+  def add_rentals(rental)
+    @rentals.push(rental)
   end
```

Comment:
- [ ] I suggest you add the logic for creating a new rental here and ensure that the `person` itself is sent as a parameter to create the new rental achieving the `has-many` association. This will be easier for you as you just need to create a book, person and date then call this method on the created person passing the book and date as the parameter. Link to the example in a previous lesson: https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship. See suggestion below:

```suggestion
  def add_rentals(book, date)
    Rental.new(date, book, self)
  end
```